Reflections/Week_1_reflection.md

```diff
@@ -0,0 +1,45 @@
+
+    * Restructure of Code (Partial)
+         
+      I have mostly restructured the code from the demo and all the comanying code into their respective files/classes. I have also began structuring how the data itself will be stored. It currently is in works to fully implement all the necessary functions and possibility of GUI being changed a bit to add more visual clairty in comparsion to the drafts/alpha build of the program. I also updated the repo to contain a small documenation markdown file with a start script that currently only runs on the linux skew. I have written down more info on enviorment and exact things required for the software to run and thinking about creating a setup script for the software for easy to use on anyone wanting to use said software. 
+
+    * Purchase of Hardware
+
+      I have purchased the preceived necessary hardware for project. Currently in shipping is a Arduino Nano 33 IoT [ABX00032] & DHT22/AM2302 Digital Temperature And Humidity Sensor Module. I discovered the sensor module during more research on hardware and it seemed like a good fit for measuring atmospheric conditions. There was some sensors specifically designed for moisture within plant soil but that didn't seem like something I wish to implement at the time.
+
+    * Breakdown of Software
+
+      I have mostly been divying up the software components into easy and digestable chunks within github and thinking of how to implement x item without simply creating one long list of code in a file. I have gotten some ideas down on my github on how to diviy up the work but still taking some thought into what exactly needs to be given an issue and what could be lumped together.
+
+2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
+   
+   I do not feel like I have any present major issues at the moment. I really haven't faced much difficulty as we head straight into development. I mostly ended up figuring out more of how data was gonna be structured and finally found the hardware to utilize within the project. Most resources I currently has sought is simple module documentation and alot of reading of module guides to get a more or less understanding of certain systems I want to implement. Also, seeing what potential future implementations could be taken.
+
+3. What do you plan to accomplish in the following week? (Please list at least 3 concrete "S.M.A.R.T." goals, along with estimated number of hours to complete, or day to complete)
+   
+   * Implement Database
+
+     I plan on implementing the basics of the database with SQL being for live data and CSV being for more arch. data. I have the SQL server up and currently been messing about with software to see if how to futher utilize it in the best manner possible. I expect it to take a few days so around 4 days to get what I want?
```

Comment:
This might be something you may have discussed in class but I think it's worth mentioning the implementation of converting SQL to CSV and where you will be storing the CSV.

Reply:
CSV was planned to be stored in CWD (Current Working Directory) of script. I was thinking about moving it to ./Documents in later reivisions for ease of access to normal user.